src/docs/data-sources.dox

```diff
@@ -26,6 +26,11 @@ cursor types that give access to data managed by WiredTiger:
 	backup cursor (optionally only returning block incremental ids if
 	<code>query_id</code> is appended),
 	key=<code>string</code>\, see @ref backup for details}
+@row{<tt>backup:export</tt>,
+	export cursor that generates a text file WiredTiger.export. The file contains metadata for all
+	objects in the database. It can be used in the import process as the value for
+	<code>metadata_file</code> configuration option,
+	key=<code>string</code>}
```

Comment:
Hmm, I think this should somehow be incorporated into the previous entry about backup cursors or split out the `query_id` piece too. Basically be consistent one way or the other.

Also all of these special cursor cases specify what the system returns that is special. So while I think it is important to tell the user about the text file, we need to make it clear what the cursor itself is returning (and what the user should do with the cursor contents).